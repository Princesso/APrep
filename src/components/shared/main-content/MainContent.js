import React from 'react';
import styled from 'styled-components'
import {Link} from 'react-router-dom'
import colors from '../../../styles/colors'
import './main-content.css'
import Button from '../button'

const MainContent = () => {
  return(
    <div className="main-content">
     <div class="caption">
        <h2>Box it! Send it!</h2>
        <p id ="desc"> Deliver parcels anywhere</p>
        <Button/>
      </div>
    </div>
  )
}

export default MainContent