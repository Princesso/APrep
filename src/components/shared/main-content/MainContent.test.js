import React from 'react';
import MainContent from './MainContent';
import renderer from 'react-test-renderer';
import {Provider} from 'react-redux';
import Store from '../../../store';
import { BrowserRouter, Route, Switch } from 'react-router-dom'

test('Registration Form', () => {
  const component = renderer.create(
    <Provider store={Store}>
      <BrowserRouter>
          <Switch>
            <MainContent/>
          </Switch>
      </BrowserRouter>
    </Provider>
  )

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
})