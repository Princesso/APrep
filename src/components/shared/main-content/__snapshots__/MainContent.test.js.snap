// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Registration Form 1`] = `
<div
  className="main-content"
/>
`;
